diff --git a/ghost/core/core/frontend/meta/schema.js b/ghost/core/core/frontend/meta/schema.js
index 894b58e429..be0978f8ea 100644
--- a/ghost/core/core/frontend/meta/schema.js
+++ b/ghost/core/core/frontend/meta/schema.js
@@ -117,6 +117,7 @@ function getHomeSchema(metaData) {
     const schema = {
         '@context': 'https://schema.org',
         '@type': 'WebSite',
+        name: escapeExpression(metaData.site.title),
         publisher: schemaPublisherObject(metaData),
         url: metaData.url,
         image: schemaImageObject(metaData.coverImage),
@@ -167,13 +168,12 @@ function getSchema(metaData, data) {
         const context = data.context ? data.context : null;
         if (_.includes(context, 'post') || _.includes(context, 'page') || _.includes(context, 'amp')) {
             return getPostSchema(metaData, data);
-        } else if (_.includes(context, 'home')) {
-            return getHomeSchema(metaData);
         } else if (_.includes(context, 'tag')) {
             return getTagSchema(metaData, data);
         } else if (_.includes(context, 'author')) {
             return getAuthorSchema(metaData, data);
         }
+        return getHomeSchema(metaData);
     }
     return null;
 }
